### DISCUSSION TITLE
Anything for the newer Quest Versions ?

### DISCUSSION THREAD

**Discussion by MiMillieuh** [QuestCraftPlusPlus/QuestCraft#409] [Q&A]
I keep my quest on the V43 because it's mentioned that QuestCraft is broken on 44 and above.

But is it still the case ? or can I upgrade to the latest version of my headset ? 

Because Background audio go brrr ! x)

**Comment by SmashHits** [comment]
yes, 3.0.0 fixed launching and 3.0.1 fixed it for more people. but the vr compatibility is different and will take awhile to get used to, also there is no mod manager
